src/getRandomNumber.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const getRandomNumber = () => {
+  const digits = [];
+
+  while (digits.length < 4) {
+    const randomDigit = Math.floor(Math.random() * (9 - 1) + 1);
```

Comment:
Change the range of random numbers from 9-1 to 9-0 to include 0 as a possible digit.